ports/fplus/vcpkg.json

```
@@ -1,6 +1,13 @@
 {
   "name": "fplus",
-  "version-semver": "0.2.14-p0",
+  "version-semver": "0.2.18-p0",
```

```suggestion
  "version": "0.2.18-p0",
```
Is this port a sermer library? If it is not, here hould use `"version"`?

---

In theory, the -p0 suffx is compliant with semver. @JonLiu1993 added the version-semver in https://github.com/microsoft/vcpkg/pull/16510 but I can't find any reference in the upstream repo that would indiciate that the project adheres to semver.

/cc @Dobiasd @JonLiu1993

---

@chausner, I went back to investigate, the modification I made a year ago was not rigorous, and I think this update should change "version-semver" to "version"

---

@chausner Yeah, FunctionalPlus does not always strictly adhere to semver. :white_check_mark: